ui/pages/notification-details/notification-details.stories.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Provider } from 'react-redux';
+import configureStore from '../../store/store';
+import NotificationDetails from './notification-details';
+import testData from '../../../.storybook/test-data';
+import { NotificationServicesController } from '@metamask/notification-services-controller';
+import { Box } from '../../components/component-library';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+  JustifyContent,
+} from '../../helpers/constants/design-system';
+import { NotificationsPage } from '../../components/multichain';
+import { Content } from '../../components/multichain/pages/page';
+import { NotificationComponents } from '../notifications/notification-components';
+import { NotificationDetailsHeader } from './notification-details-header/notification-details-header';
+import { NotificationDetailsBody } from './notification-details-body/notification-details-body';
+import { NotificationDetailsFooter } from './notification-details-footer/notification-details-footer';
+
+type Notification = NotificationServicesController.Types.INotification;
+const { processNotification } = NotificationServicesController.Processors;
```

Comment:
I kind of want us to use the new subpath exports.

In future I want us to deprecate imports like this (since they are expensive)